src/constants/wallets.ts

```diff
@@ -1,23 +1,42 @@
-import { WalletType } from 'enums';
+import { WalletType } from 'enums/wallets';
```

Comment:
WalletType is an exported enum from `enums/index`

change back to `import { WalletType } from 'enums'`